src/app/logs/_components/log-summary-header.tsx

```
@@ -0,0 +1,240 @@
+							<div className="mt-2">
+								<Modal
+									onClick={() => (window.location.href = '/configure/api/debug-zip')}
+									title="This archive may contain sensitive information"
+									wide={true}
+									body="Please inspect the contents of the zip before posting it publically. Make sure you use Moonraker Secrets if configuring moonraker for third party services."
+									content={
+										<AnimatedContainer>
+											<h3 className="mb-1 font-medium tracking-tight">The following files will be zipped</h3>
+											<ul className="grid gap-2 pb-2 text-muted-foreground">
+												<Suspense fallback={<Spinner />}>
+													{trpc.debugFileList.useSuspenseQuery()[0].map((file) => {
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)